src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,34 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
+        MEASUREMENTS_FRAMES_SLOW_RATE,
+        MEASUREMENTS_FRAMES_FROZEN_RATE,
+        MEASUREMENTS_STALL_PERCENTAGE,
+    }
+
+    def __init__(self, name, allow_array_value=False):
+        super().__init__(name)
+        self.allow_array_value = allow_array_value
```

Comment:
self.allow_array_value isn't being used, can probably be removed?

Reply:
Good point, removed!